### ISSUE TITLE
Wrong probabilities in "Reference point" tables

### ISSUE THREAD

**Issue by grinnellm** [grinnellm/HerringAssess#41]
Seems like there is a bug in the "Reference point" tables for the 3 rows with probabilities (`P(x)`) -- they have the same values for 5%, 50%, and 95% columns:
![image](https://user-images.githubusercontent.com/25674027/48434439-90e15b00-e72e-11e8-9f3c-06c4cae232b4.png)

**Comment by cgrandin** [comment]
This is because of the additions in Issue #27. This is really the wrong place to report these probabilities as they are simply a count of the B's below some value and not quantile stats (so all three are just set to the same value) but as I recall @jaclyncleary had some reason for doing so like possibly a stakeholder request.
It is satisfying to have those probabilities there beside the actual values though.

Jaclyn, can you comment before we change anything? Thx

**Comment by grinnellm** [comment]
Yes, let's wait for @jaclyncleary before changing anything. A suggestion though: I think we should move the rows with probabilities to another table -- they're not posterior percentiles, so they don't belong here. We could move them all to one table: one row per stock, and a column for each of the three probabilities. Maybe it's OK like this for this year, but we could aim to fix this for the next assessment?
